### ISSUE TITLE
Hide header, footer and sidebar in Web view

### ISSUE THREAD

**Issue by Hadharm** [AppCheap/roadmap#202]
I noticed removing header and sidebar only work for checkout. Please implement hiding header footer and side in web view too.

**Comment by Hadharm** [comment]
Or tell how to use if the feature already exist

**Comment by vantoan9708** [comment]
Hi

If you have successfully setup the **checkout page**, then you can do the same with other pages. And you just need to add this parameter to the URL to test.

`?app-builder-checkout-body-class=app-builder-checkout`
Ex:
![Screenshot 2024-06-27 at 09 23 16](https://github.com/user-attachments/assets/561db093-38da-43b6-a017-b3f70a1e7d77)

Thank you

**Comment by Hadharm** [comment]
Yes, it has been set and working perfectly in webview checkout, but it still shows in the webview widget inside the app. I am using webview widget to display some page content inside the app. why?

> Hi
> 
> If you have successfully setup the **checkout page**, then you can do the same with other pages. And you just need to add this parameter to the URL to test.
> 
> `?app-builder-checkout-body-class=app-builder-checkout` Ex: ![Screenshot 2024-06-27 at 09 23 16](https://private-user-images.githubusercontent.com/181544844/367997446-561db093-38da-43b6-a017-b3f70a1e7d77.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY2NDY0MTksIm5iZiI6MTcyNjY0NjExOSwicGF0aCI6Ii8xODE1NDQ4NDQvMzY3OTk3NDQ2LTU2MWRiMDkzLTM4ZGEtNDNiNi1hMDE3LWIzZjcwYTFlN2Q3Ny5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkxOFQwNzU1MTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hZTFjYjc4ZTIxY2I3MjRjYzllODFhYmNiMDA1M2FmZGIyYTIyYzc5MGUzYmMxNGYzYTJiYjYwMTRiMWQ1MGQ1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.bQBqAQYubN7CKSZD3Z_U6pWMpcTFiKEYjsOvkm1fyN0)
> 
> Thank you
